client/src/index.js

```diff
@@ -1,8 +1,42 @@
 import './index.css'
 import React from 'react'
 import ReactDOM from 'react-dom'
-// import Login from './components/Login/Login'
-import ListOfProjects from './components/ListOfProjects/ListOfProjects'
+import App from './App'
+import { createStore, combineReducers, applyMiddleware } from 'redux'
+import { createLogger } from 'redux-logger'
+import { Provider } from 'react-redux'
 
-// ReactDOM.render(<Login />, document.querySelector('main'))
-ReactDOM.render(<ListOfProjects/>, document.querySelector('main'))
+import createHistory from 'history/createBrowserHistory'
+
+import { ConnectedRouter, routerReducer, routerMiddleware } from 'react-router-redux'
+
+import reducers from './reducers'
+
+// Create a browser History
+const history = createHistory()
+
+// Build the middleware for intercepting and dispatching navigation actions
+const middleware = routerMiddleware(history)
```

Comment:
I would also use readable function names here than use comments. Something like the following:
```js
const browserHistory = createHistory()
const navigationMiddleware = routerMiddleware(browserHistory)
```